media-sound/qtractor/qtractor-0.9.5.ebuild

```diff
@@ -43,7 +43,7 @@ DEPEND="
 RDEPEND="${DEPEND}"
 
 PATCHES=(
-	"${FILESDIR}/${P}-dont-compress-manpages.patch"
+	"${FILESDIR}/${PN}-dont-compress-manpages.patch"
```

Comment:
why rename the patch? keep it at `0.9.4` and simply use `${PN}-0.9.4` in this line.

Reply:
I wasn't sure if there was a policy/preferred way of doing this, couldn't really find any info on it. Figured since the version isn't really relevant might as well remove the version :)
But the idea is to keep the version for which the patch was added in the patch name?
I'll update the PR to reflect this.

Reply:
in general, don't change what does not need changing ;) patch names make sense with version number they were introduced in, update whenever a new version requires rebasing of the patch.